aten/src/ATen/native/vulkan/stub/VulkanAtenStub.cpp

```diff
@@ -0,0 +1,48 @@
+#if !defined(USE_VULKAN) && !defined(USE_GLES)
```

Comment:
as mentioned earlier - better to just modify codegen (like we do for cuda) to not generate registrations if not building with vulcan

Reply:
I have added --vulkan arg to aten/gen.py the same way as --rocm, that without --vulkan no generations for Vulkan backend. 
But is_vulkan_available that is registered for CPU backend still needs this stub, so I left it here, probably that can be solved in some other place that will be enabled only when cmake USE_VULKAN